### ISSUE TITLE
--watch option doesn't seem to watch

### ISSUE THREAD

**Issue by mattsoftware** [jorgebastida/awslogs#74]
Running this command...

awslogs get --watch /var/log/php/php_errors.php ALL 

runs fine and gives me the last 1200 odd log messages. The prompt then sits there waiting, but as logs are entering into aws cloudwatchlogs, there is no hint of them in the output of the above command.

**Comment by hopperd** [comment]
I'm seeing the same issue, having the watch option doesn't appear to be properly watching.  I have to stop and re-run the command to get the latest entries.

**Comment by ticosax** [comment]
#62 is an attempt to fix this issue.

**Comment by jorgebastida** [comment]
Yes, this is a known issue related to streams getting created while the command is running. Will give a look to #62 and fix this asap.

**Comment by vpal** [comment]
Very big +1 for getting this fixed.

**Comment by TylerBrock** [comment]
Let's get this merged. Basically unusable without this feature. What's the hold up?
